fake_svc/docker-compose.yml

```diff
@@ -0,0 +1,48 @@
+services:
+  # OTEL Collector: Collects traces and logs, exports them to Jaeger and Loki
+  otel-collector:
+    image: otel/opentelemetry-collector-contrib
+    container_name: otel-collector
+    volumes:
+      - ./otel-collector-config.yaml:/etc/otelcol-contrib/config.yaml
+    ports:
+      - "4317:4317" # OTLP gRPC receiver
+      - "4318:4318" # OTLP HTTP receiver
+      - "55681:55681" # For receiving logs via the filelog receiver
+    environment:
+      - OTEL_EXPORTER_OTLP_ENDPOINT=http://otel-collector:4317  # Set endpoint to OTEL collector for traces
+      - LOGS_API_ENDPOINT=http://otel-collector:3100  # Set Loki API endpoint for logs
+
+  # Loki: Collect logs for Grafana
+  loki:
+    image: grafana/loki:2.7.3
+    container_name: loki
+    ports:
+      - "3100:3100" # Loki UI and API endpoint for logs
+    environment:
+      - LOKI_API_URL=http://otel-collector:4317 # Connect OTEL collector for logs
+
+  jaeger:
```

Comment:
Do you know how compatible it will be with Augur?

Reply:
Looks like we can define another exporter endpoint so we gucci?

Reply:
yep exactly